src/kixi/gwyn/gwyn.clj

```diff
@@ -39,19 +41,56 @@
    (-> fire-station-lookup-table
        (wds/select-from-ds {:station fire-station})
        (wds/subset-ds :cols [:radius :lat :long]))})
+
+(def google-places-api "https://maps.googleapis.com/maps/api/place/nearbysearch/json?location=")
+
+(def api-key "&key=AIzaSyB4eWT7aEhsik-rP813fNJIN-qo12nhgIM")
+
+(defn fetch-url [url]
+  (html/html-resource (java.net.URL. url)))
+
+(defn api-str [loc]
+  (str google-places-api loc api-key))
+
+(defn parse-google-places [api-call]
+  (-> api-call
+      fetch-url
+      first
+      :content
+      first
+      :content
+      first
+      json/read-str
+      (get "results")))
+
+(defn collect-property [json-entry]
+  (hash-map :address (get json-entry "vicinity")
+            :type (get json-entry "types")
+            :id (get json-entry "id")
+            :name (get json-entry "name")))
+
 (defworkflowfn list-commercial-properties-1-0-0
   {:witan/name :fire-risk/list-commercial-properties
    :witan/version "1.0.0"
    :witan/input-schema {:fire-station-geo-data sc/FireStationGeoData}
    :witan/output-schema {:commercial-properties sc/CommercialProperties}}
   [{:keys [fire-station-geo-data]} _]
-  {:commercial-properties {}})
+  {:commercial-properties
+   (let [latitude (get (ds/column fire-station-geo-data :lat) 0)
```

Comment:
I personally prefer `nth` when extracting from vectors (more idiomatic?)
Or even `first` when you're getting element 0